pages/irc.php

```diff
@@ -1,14 +1,22 @@
 <?php
 $title = __("IRC Page");
+if(!$loguserid)
+{
+  Kill("You should be log in for use the IRC.");
+}
+else
+{
+$nick = "insert default guest nick here";

```

Comment:
You can easily call loguser or something to make the name of the user come on the client. I did it, but I forgot.